lib/kennel/tasks.rb

```diff
@@ -102,7 +102,7 @@ def git_push?
 
   # also generate parts so users see and commit updated generated automatically
   desc "show planned datadog changes (scope with PROJECT=name)"
-  task plan: :generate do
+  task plan: :environment do
```

Comment:
was a bug before, called `generate` twice, did not matter since it was cached but makes the parallelism break now

Reply:
So when we merge this we need to point out to people (and add to the change notes) that `rake plan` used to write to `generated`, but now doesn't.

Reply:
Everything will still work.
`rake plan` writes to `generated` as before.
`rake generate plan` was always redundant.
It might be good to add a warning/error to anyone running `plan` and `generate` in a single command since that would be slower than just running `plan`, but only "as slow as before".

Reply:
With the PR as it stands at the moment, `rake generate plan` will (I believe) run generate, plan, and then generate again, which presumably has worse performance.

Not necessarily the end of the world; the answer might be "well don't do that then".

Or, we _consciously_ change what `plan` does, so that it _doesn't_ generate (like it used to).

(Fundamentally the problem here is that using Rake tasks is always going to make this kind of optimisation painful).